xml/System.Data/ConstraintCollection.xml

```diff
@@ -955,7 +955,7 @@
         <ReturnType>System.Collections.ArrayList</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets the list of objects contained by the collection.</summary>
```

Comment:
@joelmartinez @TianqiZhang can you please take a look?

Reply:
I'm *guessing* that this might just be more of a template issue, because if you look at the framework index file for [netcore-2.2](https://raw.githubusercontent.com/dotnet/dotnet-api-docs/master/xml/FrameworksIndex/netcore-2.2.xml), you can see this member listed as part of the `System.Data.ConstraintCollection` type:

```xml
    <Type Name="System.Data.ConstraintCollection" Id="T:System.Data.ConstraintCollection">
...
      <Member Id="P:System.Data.ConstraintCollection.List" />
    </Type>
```

mdoc is creating a member node for it because it is overridden (and thus would be a novel implementation of that member) ... `protected override System.Collections.ArrayList List { get; }`.

@TianqiZhang, do you know where that [list of members](https://docs.microsoft.com/en-us/dotnet/api/system.data.constraintcollection?view=netcore-2.2#properties) is generated from? Ideally, it would be from the list of members in the framework index file that's selected ... in this case, if `List` isn't in the list, then it would show as inherited, but then for a framework that has it overridden (as in netcore-2.2), it would show as a member implemented on that type.

Reply:
@TianqiZhang do you have an answer for Joel's question?

Reply:
@carlossanlop @mairaw @joelmartinez sorry for the late response, and yes, protected members of a sealed class are excluded when generating the yaml files. This is currently done in ECMA2Yaml as a workaround of https://github.com/mono/api-doc-tools/issues/89, we also had a mseng bug 1022811 which required us to do so.

In this `List` property case, it seems that there's a member with the same name in base class. So what's the expected behavior in such case?

Reply:
@mairaw @joelmartinez 
Should we remove the whole member?
Should we leave it undocumented?
Should we document anyway?

Reply:
Strictly speaking, once we fix the bug here (https://github.com/mono/api-doc-tools/issues/89), this member will go away ... but until then, the CI will put it back every time. As @TianqiZhang mentioned, it's already being filtered out by Ecma2Yaml as a workaround so it will never be published; so definitely don't remove it; and as for leaving it undocumented ... up to your team :)

Reply:
We don't need to document APIs that will end up disappearing. @carlossanlop do we want to keep track of them somewhere when we find those so we can remove them from our undocd efforts?

Reply:
Sure. I already have a small list.